Sources/PointFree/Episode/Show.swift

```diff
@@ -7,33 +7,36 @@ import HtmlCssSupport
 import HttpPipeline
 import HttpPipelineHtmlSupport
 import Models
+import PointFreePrelude
 import PointFreeRouter
 import Prelude
 import Styleguide
 import Tuple
 import Views
 
-let episodeResponse =
+let episodeResponse: M<Tuple5<Either<String, Episode.Id>, User?, SubscriberState, Route?, Episode.Collection.Slug?>> =
   fetchEpisodeForParam
     <| writeStatus(.ok)
     >=> userEpisodePermission
     >=> map(lower)
     >>> respond(
-      view: Views.episodeView(episodePageData:),
-      layoutData: { permission, episode, currentUser, subscriberState, currentRoute in
-        let navStyle: NavStyle = currentUser == nil ? .mountains(.main) : .minimal(.light)
+      view: newOrLegacyEpisodeView(data:),
```

Comment:
this helper view allows us to check if the current user is an admin to determine which episode to server